docker/certs/ca-key

```diff
@@ -0,0 +1,18 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
it feels wrong to commit up certs and a private key, but they are only for development use.